### PR TITLE
Run3-alca247Y Add examples for the IsoTrack calibration in Calibration/HcalCalibAlgos/macros/jobs

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Add examples for the IsoTrack calibration in Calibration/HcalCalibAlgos/macros/jobs

#### PR validation:

Used these files in getting calibration constants for 2024 IsoTrack studies

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Calibration/HcalCalibAlgos/macros/jobs/isoTrackAlCaCommissioning_cfg.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+process.options = cms.untracked.PSet( wantSummary = cms.untracked.bool(True) )
+
+process.load('RecoLocalCalo.CaloTowersCreator.calotowermaker_cfi')
+process.towerMakerAll = process.calotowermaker.clone()
+process.towerMakerAll.hbheInput = cms.InputTag("hbhereco")
+process.towerMakerAll.hoInput = cms.InputTag("none")
+process.towerMakerAll.hfInput = cms.InputTag("none")
+process.towerMakerAll.ecalInputs = cms.VInputTag(cms.InputTag("ecalRecHit","EcalRecHitsEB"), cms.InputTag("ecalRecHit","EcalRecHitsEE"))
+process.towerMakerAll.AllowMissingInputs = True
```
Comment (perrotta) [inline Calibration/HcalCalibAlgos/macros/jobs/isoTrackAlCaCommissioning_cfg.py:32]:
```suggestion
process.towerMakerAll.hbheInput = "hbhereco"
process.towerMakerAll.hoInput = "none"
process.towerMakerAll.hfInput = "none"
process.towerMakerAll.ecalInputs = [cms.InputTag("ecalRecHit","EcalRecHitsEB"), cms.InputTag("ecalRecHit","EcalRecHitsEE")]
process.towerMakerAll.AllowMissingInputs = True
```
or even (including only what modified wrt the original `calotowermaker`)
```suggestion
process.towerMakerAll.hoInput = "none"
process.towerMakerAll.hfInput = "none"
process.towerMakerAll.AllowMissingInputs = True
```

**Review by perrotta — COMMENTED** [review]
The suggestion below is just to make the config safer from possible typing errors.
In any case, I'd suggest to squeeze the commits, in order to get rid of the file remained with invalid state in the history.

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by perrotta** [comment]
@bsunanda could you also please squash the commits, to get rid of files with invalid states in the git history?

**Comment by perrotta** [comment]
@bsunanda please squash the commits, to get rid of files with invalid states in the git history

**Comment by bsunanda** [comment]
Destroyed the PR by attempting squashing. A new PR is made in #45045

**Pull request closed without merging**